src/app/page.tsx

```diff
@@ -110,13 +117,41 @@ export default function Home() {
       // user cancelled the selection
     } else {
       // user selected a single file
-      setv_correctionPaths(v_correction[0]);
+      set_v_correctionPaths(v_correction[0]);
     }
     if (DEBUG) {
       console.log("v_correction: ", v_correction);
     }
   }
 
+  async function dialogND() {
+    nd_correction = await open({
+      multiple: true})
```

Comment:
The only question I have is whether we'd like to restrict the user to only select one file for each of the calibration file dialogs, like here. I'm guessing if we take out `multiple: true`, it would only allow the user to select one file, which might make more sense. I know you said you had some ideas for FE error checking though, so just a thought :)